src/sentry/tasks/servicehooks.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import, print_function
```

Comment:
Need to add `sentry.tasks.servicehooks` into `settings.CELERY_IMPORTS`

Reply:
done